src/monitord/monitord.c

```diff
@@ -13,23 +13,102 @@
 
 /* Global variables */
 monitor_config mond;
+const char * MONTHS[] = {
+    "Jan",
+    "Feb",
+    "Mar",
+    "Apr",
+    "May",
+    "Jun",
+    "Jul",
+    "Aug",
+    "Sep",
+    "Oct",
+    "Nov",
+    "Dec"
+};
+
+time_t last_rot_log;
+time_t last_rot_json;
+
+static void rotate_logs(rotation_list *list, char *path, char *new_path, int *day, int interval, int today, int json, time_t *last_rot, time_t now) {
```

Comment:
The fuctions **rotate_logs** and **check_size_interval** are the same that the ones in agentd code. Is it possible to move them to a common place so that you don't have duplicated code?

Reply:
It's not possible due to crossed dependencies. A rework of significant Wazuh code would be needed in order to achieve this.